src/main/java/uk/gov/companieshouse/search/api/config/ElasticSearchConfig.java

```diff
@@ -7,19 +7,31 @@
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import uk.gov.companieshouse.environment.EnvironmentReader;
+import uk.gov.companieshouse.search.api.exception.EndpointException;
+
+import java.net.MalformedURLException;
+import java.net.URL;
 
 @Configuration
 public class ElasticSearchConfig {
 
     @Autowired
     private EnvironmentReader environmentReader;
 
-    private static final String HOST_NAME = "SEARCH_API_HOST";
+    private static final String END_POINT = "END_POINT";
```

Comment:
Can we add more context to the name of the environment variable here to make it clearer from the perspective of the app configuration? For example `ELASTICSEARCH_URL`.